plugins/inputs/ah_wireless/ah_wireless.go

```diff
@@ -0,0 +1,942 @@
+package ah_wireless
+
```

Comment:
Will you be providing tests?  i.e. ah_wireless_test.go

Reply:
Same question with README.md file.

Reply:
@arnelplim , I will provide test and readme once all the features are incorporated. This is initial commit. We shall have incremental commits.